src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -398,6 +418,9 @@ public void htmlPrintOptions(final PrintStream stream, final boolean printCommon
      */
     private void htmlPrintOptionTableRows(final PrintStream stream, final boolean printCommon) {
         for (final OptionDefinition optionDefinition : optionDefinitions) {
+
+            checkForNonASCII(optionDefinition.doc, optionDefinition.name);
```

Comment:
"preamble" covers both `usage` and `usageShort`. Is there any other description text?